model/model_ister.go

```diff
@@ -214,15 +215,29 @@ func JSONtoYAMLConfig(cf string) (string, error) {
 		si.Version = 0
 	}
 
-	// Remove the first occurrence of a kernel bundle and set it as the Kernel
-	for _, curr := range ic.Bundles {
-		if strings.HasPrefix(curr, "kernel-") {
-			if si.Kernel == nil {
-				si.Kernel = &kernel.Kernel{Bundle: curr}
-				continue
-			}
+	// Process the bundle information
+	// clr-installer treats the kernel differently unlike ister
+	var kernelBundles []string
+	for _, bundle := range ic.Bundles {
+		if strings.HasPrefix(bundle, "kernel-") {
+			kernelBundles = append(kernelBundles, bundle)
+			continue
+		}
+		si.Bundles = append(si.Bundles, bundle) // Set Bundles
+	}
+	// Sort the kernels as we want one that had the CDROM loadable
+	// kernel modules by default: native or LTS
+	sort.Sort(sort.Reverse(sort.StringSlice(kernelBundles)))
+	var foundKernel bool
+	for _, bundle := range kernelBundles {
+		// Only use the first kernel bundle
+		if !foundKernel {
+			foundKernel = true
+			si.Kernel = &kernel.Kernel{Bundle: bundle}
+		} else {
+			fmt.Printf("Warning: Ignoring extra kernel bundle '%s' in YAML file; '%s' already in use.\n",
```

Comment:
I have a PR in clr-installer, to skip the addtional kernel bundles if iso is set.
https://github.com/clearlinux/clr-installer/pull/604. If you feel this is a valid PR, can we avoid skipping the kernel bundles in mixer and let clr-installer handle it.